packages/kusama/src/__snapshots__/assetHubKusama.scheduler.e2e.test.ts.snap

```diff
@@ -143,7 +143,7 @@ exports[`Asset Hub Kusama Scheduler E2E tests > priority-based execution of weig
   },
   {
     "data": {
-      "when": "(rounded 29000000)",
+      "when": "(rounded 30000000)",
```

Comment:
@rockbmb this should be redacted?

Reply:
That is one mistake I made, though it hides within another: these tests are not running in post-migration Asset Hub, so there should not be a scheduler pallet available in Asset Hub.

This is running on Kusama, and not KAH.

Will remove these!